eraser-map.yaml

```diff
@@ -1325,7 +1321,7 @@ layers:
                             width: [[9, 1.0px], [10, 1.0px], [11, 1.2px], [12, 1.25px], [13, 1.5px], [14, 1.6px], [15, 0.15px], [16, 5.25px], [17, 12m]]
                             outline:
                                 color: [[13, [0.870,0.870,0.870]], [15, [0.976,0.612,0.573]]]
-                                width: [[9, 0.0px], [12, 0px], [15, 2px], [16, 2px], [17, 3px]]
+                                width: [[13, 0], [15, 2px], [16, 2px], [17, 3px]]
```

Comment:
I think the PR can land with either format (0 or 0px), as long as you link your larger Tangram issue here ;)